### DISCUSSION TITLE
Mainframe support?

### DISCUSSION THREAD

**Discussion by misterspock1** [NationalSecurityAgency/ghidra#5029] [Ideas]
Every consider adding support for 360/370/390/390x-based systems? It would be nice to have! Thanks for the great tool!

**Comment by polluks** [comment]
Right now we have just Linux subsystem's https://github.com/NationalSecurityAgency/ghidra/blob/2896551d68013e11f83862fa2db6f71579b7fcca/Ghidra/Features/Base/src/main/java/ghidra/app/util/bin/format/elf/ElfConstants.java#L195

**Reply by astrelsky** [reply]
This implies nothing

**Comment by kenrector** [comment]
Isn't  he asking about a S360 decompiler?

**Reply by polluks** [reply]
Indeed, a real first step would be to identify the exe.
Right now we have not much [EBCDIC support](https://github.com/search?q=repo%3ANationalSecurityAgency%2Fghidra%20ebcdic&type=code).
The second step is something like https://github.com/abend0c1/da
